keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,464 @@
+  * Mark enhancements "At Risk" if the following are true:
+    *  Docs PRs are not updated
+    *  The draft PRs have not been updated
+
+### Notes/Constraints/Caveats
+
+This KEP is based on a [high-level plan](https://docs.google.com/document/d/1qnfXjQCBrikbbu9F38hdzWEo5ZDBd57Cqi0wG0V6aGc/edit#) incorporating feedback collected from release team members and contributors over several cycles.
+
+In 1.22, we'll focus on implementing much of this process manually, so there will be some burden placed on KEP Authors to create the appropriate YAML structure. But ideally, these changes will free up time for people on the release team to work on automating the receipts process.
+
+### Risks and Mitigations
+
+* Non-compliance to the process by stakeholders like SIGs/KEP Authors
+  * Stricter gates have to be put for the process
+  * Exceptions only approach for a receipt to be added back to release cycle stage after the deadline
+  * Enhancement Team + Release Leads should have the final say on exceptions. The decision needs to be taken after considering the point of view of respective SIG leadership.
+* Conflicts can arise when two people update/modify the same artifact
+  * The team needs to have Git experience
+  * Only one person works on the artifacts at a time
+  * Communicate with each other when making changes
+  * Bots can eventually handle the manifest generation
+* Keeping KEP implementation history updated
+  * Requires intervention from a human and trust needs to be on the same person
+* Preventing enhancements to slip across release cycles
+* Ensuring enhancements don't permanently remain in the Beta stage
+* This KEP doesn't talk about KEP owners keeping enhancements issues up-to-date
+
+## Design Details
+
+
+### How does a Receipt look like?
+
+A Receipt has the following data stored at any point in time.
+
+```yaml
+title: Receipts process for tracking release enhancements
+kep-number: 2140
+owning-sig: sig-architecture
+path: sig-architecture/2140-receipts
+tracking-status: proposed
+stage: alpha
+```
+
+### Where are the Receipts stored?
+
+The Receipts will be stored in `releases` directory of the [k/enhancements][k-enhancements] repository. Each release will have a subdirectory and the structure would look like this.
+
+```
+tree releases
+releases
+└── v1.21
+    ├── accepted
+    │   └── 0001.yaml
+    ├── at-risk
+    │   └── 0002.yaml
+    ├── exception
+    │   └── 0003.yaml
+    ├── OWNERS
+    ├── proposed
+    │   └── 0004.yaml
+    └── removed
+        └── 0005.yaml
+```
+
+The `OWNERS` file contains the Enhancements Team for that specific Release cycle.
+
+### State Diagram
+
+![Receipts State Diagram](state-diagram.png)
+
+The flows shown in the state diagram are described in the sections below. Some of the flows described will have appropriate `kepctl` commands from the `Alpha` version itself. Other helper commands will be added and existing commands refined throughout the rollout process.
```

Comment:
In some of these flows the pull request seems to be an important mechanism for discussion and debate. For example the exception request process, or even the proposal process.

However for some of these flows the pull request seems, effectively, a rubber stamp operation. For example an enhancement subteam member accepting a KEP, or moving an accepted KEP to at-risk. What would be the  review process for these kinds of PRs? Is it expected that the enhancements team for a release will likely commit and review these internally. (i.e., one member executes `kepctl` and another approves and merges) or are even these "routine" operations intended to be points of community discussion and debate?